### PR TITLE
Handle error case in generic and echo client

### PR DESCRIPTION
**Pull request by Ram-srini**
Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

**Comment by manju956** [comment]
> Signed-off-by: Ramakrishna Srinivasamurthy [ramakrishna.srinivasamurthy@intel.com](mailto:ramakrishna.srinivasamurthy@intel.com)

CI is failing due to lint errors. Please fix them.

**Comment by manju956** [comment]
> Signed-off-by: Ramakrishna Srinivasamurthy [ramakrishna.srinivasamurthy@intel.com](mailto:ramakrishna.srinivasamurthy@intel.com)

Generic client is not covered in this PR. May be missed from the commit?

File: examples/apps/echo/client/echo_client.py [hunk 1]
```diff
@@ -356,15 +356,21 @@ def Main(args=None):
     logger.info("\n Last update to receipt receipt is:\n {}".format(
         json.dumps(receipt_update_retrieve, indent=4)
     ))
-    status = sig_obj.verify_update_receipt_signature(
-        receipt_update_retrieve['result'])
-    if status == SignatureStatus.PASSED:
-        logger.info(
-            "Work order receipt retrieve signature verification Successful")
+    if "result" in receipt_update_retrieve:
```
Comment (manojgop) [inline examples/apps/echo/client/echo_client.py:359]: So is this check present in generic client ?

File: examples/apps/generic_client/fabric_generic_client.py [hunk 2]
```diff
@@ -604,9 +604,13 @@ def Main(args=None):
                 wo_receipt,
                 wo_params, jrpc_req_id)
         # Verify receipt signature
-        if generic_client.verify_receipt_signature(
-                retrieve_wo_receipt) is False:
-            logger.error("Receipt signature verification Failed")
+        if "result" in retrieve_wo_receipt:
```
Comment (manojgop) [inline examples/apps/generic_client/fabric_generic_client.py:607]: Is this check applicable only to fabric_generic_client and not required for generic_client direct model ?

Comment (Ram-srini) [reply examples/apps/generic_client/fabric_generic_client.py:607]: required for generic_client in direct model as well.

File: examples/apps/echo/client/echo_client.py (hunk 1 above)
Comment (Ram-srini) [reply examples/apps/echo/client/echo_client.py:359]: yes

**Pull request merged**